.github/workflows/build-docs.yml

```diff
@@ -15,17 +15,16 @@ jobs:
 
     - name: Setup node
       uses: actions/setup-node@v1
```

Comment:
I know this was out of date before you touched it but maybe we could move this to v2 of the action while we are here?

Reply:
I hadn't thought to check if there was a v2. No reason not to do this.

Reply:
Could also do the same for `actions/setup-java@v1` if you want?